artifacts/README.md

```diff
@@ -0,0 +1 @@
+Ephemeral artifacts are saved to this directory.
```

Comment:
Ha, oops. I didn't mean to commit this. It was replaced by the `flow` directory.